### DISCUSSION TITLE
Serialization of Containment

### DISCUSSION THREAD

**Discussion by grafandreas** [eclipse-emfcloud/emfcloud#191] [EMFJson Jackson] [answered]
Dear Team,

is there a possibility to configure the (de) serializer, so that containment references are serialized as actual references, and not have the containment hierarchy in one document? (w/o modifying the ecore).

Thanks,

Andreas

**Comment by eneufeld** [comment, marked as answer]
HI @grafandreas ,
yes you could override https://github.com/eclipse-emfcloud/emfjson-jackson/blob/ed9069ec6c80bc7ddc276c2e6888aee24cb9c888/src/main/java/org/eclipse/emfcloud/jackson/module/EMFModule.java#L172 and there use a custom version of `EMFDeserializers` and `EMFSerializers` .

Just out of curiosity why do you want to split the tree to multiple files?
